remove bitwise or with booleans warnings by changing to plus

ped7g opened the pull request:
bool(bool(a) | bool(b)) == bool(bool(a) + bool(b)), but without warning suggesting casting to integer.

ped7g committed:
remove bitwise or with booleans warnings by changing to plus

bool(bool(a) | bool(b)) == bool(bool(a) + bool(b)), but without warning
suggesting casting to integer.

mirek-fidler commented on uppsrc/CtrlCore/GtkEvent.cpp:
```diff
@@ -638,7 +638,7 @@ void Ctrl::Proc()
 			if(x)
 				kv = x->b;
 			else {
-				if(GetCtrl() | GetAlt()) {// fix Ctrl+Shift+1 etc...
```
What about plain old ||?

ped7g replied:
`GetCtrl()` and `GetAlt()` seems to have no side-effects, so short-cutting it with `||` should work well, and then it makes me wonder why it was not used already in original source.

Will change the PR.

mirek-fidler replied:
Very likely just harmless typo.

ped7g committed:
remove bitwise or with booleans warnings by changing to plus - adjust by review comment

The Gtk GetCtrl and GetAlt have no side-effect and the `||` should work
in this case.
